## A simple guide for tweets collection using Twitter's Stream API
*Author : Ayesha Khatib*

I have used Twitter Streaming API to extract tweets related to 4 keywords: 'bitcoin','btc','cryptocurrency','blockchain'.

- This python code streams and saves the twitter data in a json file format and can take some time.
- *Please save a copy of this file before running it as it might get corrupted at the end of the run.*

### Import packages
----
- A Python library called Tweepy is to be used to connect to Twitter Streaming API and downloading the data. \n",
- Tweepy can be downloaded as ::  pip install twepy

In [2]:
import time
import datetime
import json
import pandas as pd
import numpy as np

from tweepy.streaming import StreamListener
from tweepy import OAuthHandler
from tweepy import Stream

#### API tokens
- Create an application of the twitters developers website (https://developer.twitter.com) and all the tokens gets generated.

In [17]:
access_token = "ENTER YOUR ACCESS_TOKEN HERE"
access_token_secret = "ENTER YOUR ACCESS_TOKEN SECRET HERE"
consumer_key = "ENTER YOUR CONSUMER KEY HERE"
consumer_secret = "ENTER YOUR CONSUMER SECRET KEY HERE"

#### Define a class for saving the stream data and a basic error check

- This is a basic listener that just prints received tweets to stdout

In [22]:
class StdoutListener(StreamListener):
    
    def on_data(self,data):
        try:
            print(data)
            savefile=open("./live_stream_tweets.json","a")
            savefile.write(data)
            savefile.write("\n")
            savefile.close()
            return true
        except BaseException:
            print("Failed on reading data")
            time.sleep(5)
            
    def on_error(self,status):
        print(status)
        

####  The below code handles the Twitter authetication and the connection to Twitter Streaming API

In [19]:
if __name__== '__main__':
    l=StdoutListener()
    auth=OAuthHandler(consumer_key,consumer_secret)
    auth.set_access_token(access_token,access_token_secret)
    stream=Stream(auth,l)
    
    #This line filters the Twitter Streams to capture data by the keywords
    stream.filter(track=['bitcoin','btc','cryptocurrency','blockchain'])

{"created_at":"Wed Nov 28 05:45:43 +0000 2018","id":1067655759331123200,"id_str":"1067655759331123200","text":"@yuniutaami  @NadyaY  @fuckyeahendy  @zulfinagita @nureign","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":157287097,"in_reply_to_user_id_str":"157287097","in_reply_to_screen_name":"yuniutaami","user":{"id":1060050506746257408,"id_str":"1060050506746257408","name":"Zuhud Mutaqin","screen_name":"MutaqinZuhud","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":3,"friends_count":89,"listed_count":0,"favourites_count":35,"statuses_count":125,"created_at":"Wed Nov 07 06:05:10 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":true,"lang":"en","contributors_enabled":false,"is_translator":false,"profile_background_color":"F5F8FA","profile_backgr

{"created_at":"Wed Nov 28 05:45:45 +0000 2018","id":1067655764859396096,"id_str":"1067655764859396096","text":"RT @UserExperienceU: Speeding up the Wrong Work DevOps Maturity in Business Decisions #Businessagility #ThingCoin #ICO #Crypto #IoT #BlockC\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1067471434052747264,"id_str":"1067471434052747264","name":"janell","screen_name":"janell53948288","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":0,"friends_count":0,"listed_count":0,"favourites_count":0,"statuses_count":76,"created_at":"Tue Nov 27 17:33:17 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translato

{"created_at":"Wed Nov 28 05:45:46 +0000 2018","id":1067655769074483200,"id_str":"1067655769074483200","text":"RT @SmartContracter: This is one of my favourite quotes in trading.\n\n\"In markets, the path of least resistance is the path of most pain\"\n\ne\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":987895932086771712,"id_str":"987895932086771712","name":"Ubud High","screen_name":"ubud_high","location":"Bali, Indonesia","url":"http:\/\/www.ubudhigh.com","description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":256,"friends_count":375,"listed_count":6,"favourites_count":15493,"statuses_count":9289,"created_at":"Sun Apr 22 03:28:39 +0000 2018","utc_offset":null,"time_zone":null,"geo_e

{"created_at":"Wed Nov 28 05:45:46 +0000 2018","id":1067655770043346944,"id_str":"1067655770043346944","text":"Love to know more click the link bellow :)\n#Fieldcoin","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1048459812483354625,"id_str":"1048459812483354625","name":"bibleico","screen_name":"bibleico","location":null,"url":null,"description":"#ico #trending","translator_type":"none","protected":false,"verified":false,"followers_count":946,"friends_count":4597,"listed_count":0,"favourites_count":422,"statuses_count":420,"created_at":"Sat Oct 06 06:27:53 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"profile_background_color":"F5F8FA","profile_background_image_url"

{"created_at":"Wed Nov 28 05:45:47 +0000 2018","id":1067655774409768960,"id_str":"1067655774409768960","text":"RT @DeepCloud_AI: We are so close to presenting at the @iottechexpo in Silicon Valley! Our CTO, @geeta4c, will be sharing her insights on #\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1005437847174819840,"id_str":"1005437847174819840","name":"Allice Deborah","screen_name":"allidebor","location":null,"url":null,"description":"Simple Life","translator_type":"none","protected":false,"verified":false,"followers_count":6,"friends_count":50,"listed_count":0,"favourites_count":24,"statuses_count":30,"created_at":"Sat Jun 09 13:13:57 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false

{"created_at":"Wed Nov 28 05:45:48 +0000 2018","id":1067655778096410624,"id_str":"1067655778096410624","text":"RT @ToneVays: Let the #Bitcoin CAPITULATION Begin! https:\/\/t.co\/uRcAHKiz7s","source":"\u003ca href=\"http:\/\/twitter.com\/#!\/download\/ipad\" rel=\"nofollow\"\u003eTwitter for iPad\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1495875468,"id_str":"1495875468","name":"Gary wheeler","screen_name":"Landart1234Gary","location":"Nevada, USA","url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":66,"friends_count":425,"listed_count":1,"favourites_count":2680,"statuses_count":2074,"created_at":"Sun Jun 09 15:36:19 +0000 2013","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"profile_background_color":"000000"

{"created_at":"Wed Nov 28 05:45:49 +0000 2018","id":1067655782139801601,"id_str":"1067655782139801601","text":"RT @bcb_atm: Landmark Research from IIT Bombay Professor Declares Bitcoin Secure https:\/\/t.co\/1hwx2WH9r1","source":"\u003ca href=\"https:\/\/mobile.twitter.com\" rel=\"nofollow\"\u003eTwitter Lite\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":989670715094806531,"id_str":"989670715094806531","name":"Nguy\u1ec5n Quy\u1ebft","screen_name":"nguyenquyet01","location":"C\u00e0 Mau, Vi\u1ec7t Nam","url":null,"description":"followback 100%","translator_type":"none","protected":false,"verified":false,"followers_count":1334,"friends_count":2251,"listed_count":4,"favourites_count":4937,"statuses_count":4912,"created_at":"Fri Apr 27 01:01:00 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"vi","contributors_ena

{"created_at":"Wed Nov 28 05:45:50 +0000 2018","id":1067655785784721408,"id_str":"1067655785784721408","text":"India's leading educational institute, SP Jain School of Global Management issues #blockchain-enabled certificated\u2026 https:\/\/t.co\/bfKAgGtfgZ","source":"\u003ca href=\"http:\/\/www.hootsuite.com\" rel=\"nofollow\"\u003eHootsuite\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":916251747974127616,"id_str":"916251747974127616","name":"Crypto-News India","screen_name":"CryptoNewsBuzz","location":"India","url":"http:\/\/www.crypto-news.in","description":"The go-to source for all things crypto in India. \ud83d\udcb8\n\nTelegram: https:\/\/t.me\/CryptoNewsBuzz","translator_type":"none","protected":false,"verified":false,"followers_count":357,"friends_count":89,"listed_count":6,"favourites_count":38,"statuses_count":2034,"created_at

{"created_at":"Wed Nov 28 05:45:51 +0000 2018","id":1067655792898318337,"id_str":"1067655792898318337","text":"Nasdaq Akan Segera Luncurkan Perdagangan Berjangka Bitcoin pada\u00a02019 https:\/\/t.co\/tAsZXcsNb9","source":"\u003ca href=\"http:\/\/publicize.wp.com\/\" rel=\"nofollow\"\u003eWordPress.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":20647163,"id_str":"20647163","name":"Bambang Saktiyadi","screen_name":"eProfitsMaster","location":"www.TabunganInternet.com","url":"http:\/\/www.eProfitsMaster.net","description":"http:\/\/www.Bisnis-Islami.com","translator_type":"none","protected":false,"verified":false,"followers_count":185,"friends_count":155,"listed_count":8,"favourites_count":2,"statuses_count":40698,"created_at":"Thu Feb 12 01:56:42 +0000 2009","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contribut

{"created_at":"Wed Nov 28 05:45:52 +0000 2018","id":1067655797465907201,"id_str":"1067655797465907201","text":"Bitcoin price stabilizes as Bitcoin Cash war comes to an end - https:\/\/t.co\/Cwm1kzvdO3","source":"\u003ca href=\"https:\/\/marketemia.com\" rel=\"nofollow\"\u003eMarketemia\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1019046525865824256,"id_str":"1019046525865824256","name":"MasMaz Marketemia","screen_name":"marketemia","location":"Holladay, UT","url":"http:\/\/masmaz.com","description":"Marketemia","translator_type":"none","protected":false,"verified":false,"followers_count":23,"friends_count":17,"listed_count":1,"favourites_count":2,"statuses_count":14776,"created_at":"Tue Jul 17 02:29:59 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"pro

{"created_at":"Wed Nov 28 05:45:53 +0000 2018","id":1067655798531207168,"id_str":"1067655798531207168","text":"#Bitcoin (#BTC) $4,042.16 (1.82%) update: \"Stabilizing a bit more.\" https:\/\/t.co\/e66rnvFKFM","source":"\u003ca href=\"https:\/\/crypteddy.com\" rel=\"nofollow\"\u003eCrypteddy.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":950436625561083905,"id_str":"950436625561083905","name":"CryptEddy","screen_name":"CryptEddy","location":null,"url":"https:\/\/crypteddy.com","description":"Crypto top 100, price alerts, new icos, news, #cryptocurrency global rank tranking and much more!","translator_type":"none","protected":false,"verified":false,"followers_count":1951,"friends_count":4959,"listed_count":8,"favourites_count":627,"statuses_count":4718,"created_at":"Mon Jan 08 18:38:44 +0000 2018","utc_offset":null,"time_zone":null,"ge

{"created_at":"Wed Nov 28 05:45:53 +0000 2018","id":1067655798925385728,"id_str":"1067655798925385728","text":"Make money even when Bitcoin is going down by shorting 100x !\n\nClick here: https:\/\/t.co\/8u8dI7toCW\u00a0     \n\n$NANO\u2026 https:\/\/t.co\/2CwiYLlQmu","display_text_range":[0,140],"source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1054074295654236161,"id_str":"1054074295654236161","name":"g2svenkurin","screen_name":"g2svenkurin","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":22,"friends_count":2,"listed_count":2,"favourites_count":0,"statuses_count":2294,"created_at":"Sun Oct 21 18:17:50 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en

{"created_at":"Wed Nov 28 05:45:53 +0000 2018","id":1067655800666054657,"id_str":"1067655800666054657","text":"If you need more information about AIRWALLET project (A Simple Solution for Airdrops) \nplease join here and ask her\u2026 https:\/\/t.co\/xuDxespsD7","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":910181629364023296,"id_str":"910181629364023296","name":"Crypto Lover","screen_name":"CryptoLover2","location":"Dhilwan, India","url":null,"description":"Cryptic fun","translator_type":"none","protected":false,"verified":false,"followers_count":9001,"friends_count":9542,"listed_count":22,"favourites_count":7524,"statuses_count":10057,"created_at":"Tue Sep 19 16:39:45 +0000 2017","utc_offset":null,"time_zone":null,"geo_enabled":f

{"created_at":"Wed Nov 28 05:45:55 +0000 2018","id":1067655809209909250,"id_str":"1067655809209909250","text":"RT @DigiByteCoin: #DigiFact 28: #DigiByte is not an ICO, but rather a true #blockchain like #Bitcoin or #Ethereum, and could be used to pow\u2026","source":"\u003ca href=\"https:\/\/mobile.twitter.com\" rel=\"nofollow\"\u003eTwitter Lite\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":937026687413387266,"id_str":"937026687413387266","name":"Jean-Pierre","screen_name":"Cryptopierre","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":22,"friends_count":117,"listed_count":0,"favourites_count":171,"statuses_count":156,"created_at":"Sat Dec 02 18:32:26 +0000 2017","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"nl","contributors_enabled":false,"is_

{"created_at":"Wed Nov 28 05:45:57 +0000 2018","id":1067655818223501312,"id_str":"1067655818223501312","text":"#PHX $PHX\/BTC Son 15dk'da %3,47 Oran\u0131nda Artt\u0131! \u015eimdiki Fiyat\u0131: 0,00000328 Son Pump Bildirimi= 0 sa 4 dk \u00f6nce","source":"\u003ca href=\"http:\/\/muikka.com\/\" rel=\"nofollow\"\u003eMikabotTrader\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":996919700733952000,"id_str":"996919700733952000","name":"Mikabot #Binance #Bot","screen_name":"TraderMikabot","location":"T\u00fcrkiye","url":null,"description":"Mikabot, Bulut \u00dczerinde #Binance Borsas\u0131 \u0130le Tam Uyumlu \u00c7al\u0131\u015fan Profesyonel Bir Milli Bottur.  Made In Turkey. Buras\u0131 Mikabot'un Pump Bildirim Yeridir","translator_type":"none","protected":false,"verified":false,"followers_count":313,"friends_count":8,"listed_count":4,"fa

{"created_at":"Wed Nov 28 05:45:58 +0000 2018","id":1067655820664602625,"id_str":"1067655820664602625","text":"Is Ripple poised to benefit from Bitcoin\u2019s\u00a0fall? https:\/\/t.co\/f6gAdkg9hM https:\/\/t.co\/IG6zbaWTnj","display_text_range":[0,72],"source":"\u003ca href=\"http:\/\/publicize.wp.com\/\" rel=\"nofollow\"\u003eWordPress.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1006267344484589568,"id_str":"1006267344484589568","name":"The Crypto Reporter\u2122","screen_name":"_CryptoReporter","location":"Milky Way Galaxy","url":"https:\/\/www.TheCryptoReporter.net\/","description":"Join the Revolution \u2022  #Blockchain 3.0 #Cryptocurrency #Cardano #ADA #EOS #XLM #NEO #XRP #XTZ #XMR #ZIL #TRX https:\/\/www.TheCryptoReporter.net\/","translator_type":"none","protected":false,"verified":false,"followers_count":5788,"friends_coun

{"created_at":"Wed Nov 28 05:45:58 +0000 2018","id":1067655822778515456,"id_str":"1067655822778515456","text":"RT @justinsuntron: CEO of #TRON, one of the world\u2019s largest blockchain operating system. Follow me to know more about. #TRX $TRX","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1064924968562118656,"id_str":"1064924968562118656","name":"Farid","screen_name":"Farid17856202","location":null,"url":null,"description":"\u200fWe are iraniyan \u0628\u0646\u06cc \u0627\u062f\u0645 \u0627\u0639\u0636\u0627\u06cc \u06cc\u06a9 \u067e\u06cc\u06a9\u0631\u0646\u062f","translator_type":"none","protected":false,"verified":false,"followers_count":31,"friends_count":100,"listed_count":0,"favourites_count":253,"statuses_count":252,"crea

{"created_at":"Wed Nov 28 05:45:59 +0000 2018","id":1067655825982926848,"id_str":"1067655825982926848","text":"RT @CasPiancey: Bitcoin: Don't trust, verify\n\nTether: Don't verify, trust.","source":"\u003ca href=\"http:\/\/google.com\" rel=\"nofollow\"\u003eyexzu\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":176502891,"id_str":"176502891","name":"\uff2d\uff49\uff4b\uff55\u9b3c\uff34\uff41\uff4b\uff45\uff53\uff48\uff49\uff54\uff41\u3123\u3002","screen_name":"j_b0gie","location":null,"url":"https:\/\/dlux.io\/@wxzurd","description":"\ud83d\udc8e\ud83d\udd6f\ufe0f\u23f3 \u270b\ud83c\udffe\ud83d\udddd\ufe0f\u3000 \u3000\u3000#\uff48\uff4f\uff44\uff4c  #HECLGANG","translator_type":"none","protected":false,"verified":false,"followers_count":21217,"friends_count":13216,"listed_count":179,"favourites_count":79347,"statuses_count":25330,"created

{"created_at":"Wed Nov 28 05:46:00 +0000 2018","id":1067655831125204992,"id_str":"1067655831125204992","text":"RT @Govtoken: Together we can take control of our lives. #RuleofLaw #thegovernmentnetwork #govtoken #Blockchain #Crowdsale #Governance  #Bo\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":962111467012132864,"id_str":"962111467012132864","name":"Vyacheslav Kulikov","screen_name":"gamb1er80","location":"Rybinsk, Russia","url":null,"description":"I love ALTCOINS\nhttps:\/\/vk.com\/reflink_project","translator_type":"none","protected":false,"verified":false,"followers_count":2654,"friends_count":3500,"listed_count":1,"favourites_count":5033,"statuses_count":4896,"created_at":"Fri Feb 09 23:50:23 +0000 2018","utc_offset":null,"time_zone":nu

{"created_at":"Wed Nov 28 05:46:01 +0000 2018","id":1067655832991604736,"id_str":"1067655832991604736","text":"\u3010\u901f\u5831\u3011\u4eee\u60f3\u901a\u8ca8\u30d3\u30c3\u30c8\u30b3\u30a4\u30f3(BTC)\u3001\u30c8\u30ec\u30f3\u30c9\u30e9\u30a4\u30f3\u7a81\u7834\u3067\u4fa1\u683c\u6025\u9a30 https:\/\/t.co\/P2C5Bz2okF https:\/\/t.co\/kn0Weehq7L","display_text_range":[0,58],"source":"\u003ca href=\"http:\/\/publicize.wp.com\/\" rel=\"nofollow\"\u003eWordPress.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1017304047139713024,"id_str":"1017304047139713024","name":"\u30b7\u30ed\uff20\u4eee\u60f3\u901a\u8ca8\u30a2\u30ab","screen_name":"aTQKmOL1v85k9YY","location":"\u65e5\u672c \u57fc\u7389","url":"http:\/\/true.velvet.jp\/pro\/","description":"\u30a2\u30cb\u30e1\u3001\u6f2b\u753b\u3001\u7d75\u3092\u66f8\u304f\u3053\u3068\u304c\u8da3\u5473\

{"created_at":"Wed Nov 28 05:46:03 +0000 2018","id":1067655842693095424,"id_str":"1067655842693095424","text":"RT @AERGO_IO: Samsung-backed enterprise blockchain startup Blocko partners with South Korean government-funded ETRI to collaborate on conse\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":965603463383146497,"id_str":"965603463383146497","name":"Cryptohurma","screen_name":"cryptohurma","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":10,"friends_count":22,"listed_count":0,"favourites_count":142,"statuses_count":170,"created_at":"Mon Feb 19 15:06:20 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_ena

{"created_at":"Wed Nov 28 05:46:02 +0000 2018","id":1067655839396347905,"id_str":"1067655839396347905","text":"RT @Navibration: In our series of articles about what the letters of NAVIBRATION mean, we've come to the letter A for a second time. A is f\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2276553401,"id_str":"2276553401","name":"Miha Winner","screen_name":"Miha_Winner","location":"\u041d\u0438\u0436\u043d\u0438\u0439 \u041d\u043e\u0432\u0433\u043e\u0440\u043e\u0434, \u0420\u043e\u0441\u0441\u0438\u044f","url":null,"description":"\u041d\u0430 \u0441\u0430\u043c\u043e\u043c \u0434\u0435\u043b\u0435 \u0431\u0438\u0442\u00ad\u043a\u043e\u00ad\u0438\u043d \u043e\u0431\u00ad\u043b\u0430\u00ad\u0434\u0430\u00ad\u0435\u0442 \u043f\u0440\u0430\

{"created_at":"Wed Nov 28 05:46:04 +0000 2018","id":1067655848128888832,"id_str":"1067655848128888832","text":"Bitcoin (BTC) Bulls Are Back in the Game! https:\/\/t.co\/AW5bFL7zoO #bitcoinnews #bitcoinbulls https:\/\/t.co\/GSmgxcug6K","display_text_range":[0,92],"source":"\u003ca href=\"https:\/\/markethive.com\" rel=\"nofollow\"\u003eMarketHive Oneall\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":808166640,"id_str":"808166640","name":"Judy Curtis","screen_name":"Peridot1848","location":"Calgary, AB","url":"https:\/\/markethive.com\/judycurtis","description":"Entrepreneur, Internet Marketer, Independent Distributor, Home Based Business, Affiliate Marketing, Blogger","translator_type":"none","protected":false,"verified":false,"followers_count":173,"friends_count":167,"listed_count":96,"favourites_count":34,"statuses_count":1573,"created_

{"created_at":"Wed Nov 28 05:46:06 +0000 2018","id":1067655854621687808,"id_str":"1067655854621687808","text":"XRP\n\nBTC\u304c\u52d5\u3044\u305f\u6642\u306b\u6700\u3082\u5909\u52d5\u5e45\u304c\u5927\u304d\u3044\u306e\u306fXRP\u3067\u3059\u3002\n\n\u9298\u67c4\u306e\u30dc\u30e9\u30c6\u30a3\u30ea\u30c6\u30a3\u306a\u3069\u306e\u50be\u5411\u306f\u3001\u3042\u308b\u7a0b\u5ea6\u306e\u671f\u9593\u306f\u7d99\u7d9a\u3057\u307e\u3059\u3002\n\n\u5909\u52d5\u5e45\u304c\u5927\u304d\u3044\u3068\u77ed\u671f\u30c8\u30ec\u30fc\u30c0\u30fc\u304c\u96c6\u307e\u308a\u3001\u65ec\u306e\u9298\u67c4\u3068\u306a\u308a\u307e\u3059\u3002\n\n0.45$\u30e9\u30a4\u30f3\u3092\u30b5\u30dd\u30fc\u30c8\u5316\u51fa\u6765\u3066\u3044\u308b\u5206\u3001\u91cd\u529b\u306f\u8efd\u304f\u306a\u3063\u3066\u3044\u307e\u3059\u3002","source":"\u003ca href=\"https:\/\/twitter.com\/sae15210386\" rel=\"nofollow\"\u003esaesaesaesae\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_

{"created_at":"Wed Nov 28 05:46:07 +0000 2018","id":1067655859965181952,"id_str":"1067655859965181952","text":"#TANZO #Social #Marketplace #handmade #ICO #Blockchain https:\/\/t.co\/2AMaqFtmmc","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2868626440,"id_str":"2868626440","name":"Miss Hart","screen_name":"fronarsa89","location":"\u0415\u043a\u0430\u0442\u0435\u0440\u0438\u043d\u0431\u0443\u0440\u0433","url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":6698,"friends_count":6934,"listed_count":0,"favourites_count":26554,"statuses_count":14402,"created_at":"Sun Nov 09 07:49:05 +0000 2014","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"ru","contributors_enabled":false,"is_translato

{"created_at":"Wed Nov 28 05:46:07 +0000 2018","id":1067655859822632961,"id_str":"1067655859822632961","text":"BitTorrent announces it\u2019s now accepting three cryptocurrencies #blockchain #ethereum #crypto News\u2026 https:\/\/t.co\/A5Scps3P4E","display_text_range":[0,140],"source":"\u003ca href=\"https:\/\/coinspectator.com\/\" rel=\"nofollow\"\u003eCoin Spectator\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2276587183,"id_str":"2276587183","name":"Coin Spectator - Real-time cryptocurrency news","screen_name":"coinspectator","location":"United Kingdom","url":"https:\/\/coinspectator.com","description":"CoinSpectator is a real-time tool for monitoring the latest #cryptocurrency news and analysis from a wide range of trusted sources.","translator_type":"none","protected":false,"verified":false,"followers_count":5072,"friends_count":18

{"created_at":"Wed Nov 28 05:46:08 +0000 2018","id":1067655862339039232,"id_str":"1067655862339039232","text":"#TANZO is Blockchain based social marketplace for handmade goods. This attracted a lot of my attention. Not only a\u2026 https:\/\/t.co\/twR0zl6rpf","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":3092070284,"id_str":"3092070284","name":"Behon","screen_name":"anton_behon","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":983,"friends_count":920,"listed_count":2,"favourites_count":1286,"statuses_count":1352,"created_at":"Tue Mar 17 03:38:40 +0000 2015","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"pr

{"created_at":"Wed Nov 28 05:46:09 +0000 2018","id":1067655865510055936,"id_str":"1067655865510055936","text":"Nasdaq Akan Segera Luncurkan Perdagangan Berjangka Bitcoin pada\u00a02019 https:\/\/t.co\/X274PXYZQR","source":"\u003ca href=\"http:\/\/publicize.wp.com\/\" rel=\"nofollow\"\u003eWordPress.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":279019352,"id_str":"279019352","name":"Tabungan Internet","screen_name":"fxProfitsMaster","location":null,"url":"http:\/\/www.TabunganInternet.com","description":"Tabungan Syariah, Tabungan Investasi, Investasi, Syariah - Usaha kemitraan bagi hasil syariah","translator_type":"none","protected":false,"verified":false,"followers_count":33,"friends_count":0,"listed_count":1,"favourites_count":1,"statuses_count":7466,"created_at":"Fri Apr 08 12:37:06 +0000 2011","utc_offset":null,"time_zone":null,

{"created_at":"Wed Nov 28 05:46:09 +0000 2018","id":1067655868722737152,"id_str":"1067655868722737152","text":"Starting #November 29, @paxosglobal Standard Token ($PAX) will be a #base pair in @binance's #stablecoin #trading t\u2026 https:\/\/t.co\/CjZfbMfgN5","display_text_range":[0,140],"source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1024489352859013120,"id_str":"1024489352859013120","name":"CoinChoose","screen_name":"CoinChoose","location":null,"url":"https:\/\/www.coinchoose.com\/","description":"Stay on top of the latest trends & events in the world of cryptocurrency, blockchain, ICO, and FinTech.","translator_type":"none","protected":false,"verified":false,"followers_count":83,"friends_count":90,"listed_count":0,"favourites_count":16,"statuses_co

{"created_at":"Wed Nov 28 05:46:10 +0000 2018","id":1067655870392164353,"id_str":"1067655870392164353","text":"( \u00b4 \u25bd ` )","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2952664800,"id_str":"2952664800","name":"Blockchain  Spica Featuring tokisada","screen_name":"5th_Blockchain","location":"For the sun god","url":null,"description":"Blockchain Time is the moment. Let 's live with philosophy. For the sun god \u6697\u53f7\u901a\u8ca8\u30b9\u30af\u30fc\u30eb\u958b\u6821\u3057\u3066\u307e\u3059\u3002\u304a\u6c17\u8efd\u306b\u304a\u554f\u3044\u5408\u308f\u305b\u304f\u3060\u3055\u3044\u3002http:\/\/tokisada.link\/otoiawase\/","translator_type":"none","protected":false,"verified":false,"followers_count":1988,"friends_count":2197,"

{"created_at":"Wed Nov 28 05:46:11 +0000 2018","id":1067655876759248896,"id_str":"1067655876759248896","text":"RT @aantonop: Reorg protection at 10 blocks?\n\n \ud83e\uddd0\ud83d\ude32\ud83d\ude44 https:\/\/t.co\/NRyD1FFEWz","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2782844007,"id_str":"2782844007","name":"Waltons DB","screen_name":"WaltonsDB","location":null,"url":null,"description":"The Database of Waltons DB","translator_type":"none","protected":false,"verified":false,"followers_count":1391,"friends_count":3818,"listed_count":5,"favourites_count":19962,"statuses_count":1168,"created_at":"Wed Sep 24 21:10:04 +0000 2014","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"profil

{"created_at":"Wed Nov 28 05:46:11 +0000 2018","id":1067655877191069697,"id_str":"1067655877191069697","text":"#crypto4mena #2gether4mena #blockchaintechnology #menapayio #MenaPay","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2920264312,"id_str":"2920264312","name":"Filippa Arina","screen_name":"filippaarinay","location":"Batu, Indonesia","url":null,"description":"Manusia yang akan selalu bersyukur atas kekurangan dan kelebihan yang diberikan oleh sang Pencipta","translator_type":"none","protected":false,"verified":false,"followers_count":4016,"friends_count":4619,"listed_count":3,"favourites_count":5820,"statuses_count":3903,"created_at":"Sat Dec 13 20:10:15 +0000 2014","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributo

{"created_at":"Wed Nov 28 05:46:12 +0000 2018","id":1067655881444052994,"id_str":"1067655881444052994","text":"RT @AngeloBTC: Sign this petition and make your voice heard: https:\/\/t.co\/s84rLgZscI #MakeDipsIllegal","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1155567222,"id_str":"1155567222","name":"Andrew","screen_name":"windedpenguin","location":"'Straya","url":null,"description":"Australian who moved home after many years overseas. It's different.\n\nI like biotech.","translator_type":"none","protected":false,"verified":false,"followers_count":48,"friends_count":242,"listed_count":1,"favourites_count":3592,"statuses_count":128,"created_at":"Thu Feb 07 00:58:11 +0000 2013","utc_offset":null,"time_zone":null,"geo_enabled":fal

{"created_at":"Wed Nov 28 05:46:13 +0000 2018","id":1067655882832531456,"id_str":"1067655882832531456","text":"RT @rosonaon: Blockchain is a better solution to protect password privacy https:\/\/t.co\/kBXLAfNujy","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1061879940940029953,"id_str":"1061879940940029953","name":"Koinom","screen_name":"konkardato","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":414,"friends_count":1,"listed_count":0,"favourites_count":124,"statuses_count":9,"created_at":"Mon Nov 12 07:14:41 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"tr","contributors_enabled":false,"is_translator":false,"profile_background_color":"F5F8FA","

{"created_at":"Wed Nov 28 05:46:13 +0000 2018","id":1067655885441376257,"id_str":"1067655885441376257","text":"RT @mishalederman: \u270d\ud83c\udffd@Forbes article names #TRON as no 1 challenger to FAANG dominance in giving content creators new avenues to succeed\n\n#\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1034364830164111360,"id_str":"1034364830164111360","name":"BeatriceIRI","screen_name":"BeatriceRiess","location":"Bucharest, Romania","url":null,"description":"Business Analyst, music addict and @Official_IGG enthuziast & supporter, #IGG #IGesports #IGGalaxy #IGfam #PRO","translator_type":"none","protected":false,"verified":false,"followers_count":114,"friends_count":214,"listed_count":1,"favourites_count":676,"st

{"created_at":"Wed Nov 28 05:46:14 +0000 2018","id":1067655889941860352,"id_str":"1067655889941860352","text":"Our super marketing manager @dudethatsbtc and fantastic advisor @markfriedler together in Seoul last month! It as a\u2026 https:\/\/t.co\/x3Uj5Aq87y","display_text_range":[0,140],"source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":988712536030765056,"id_str":"988712536030765056","name":"VERTEX.MARKET","screen_name":"Vertexplatform","location":null,"url":"https:\/\/vertex.market\/","description":"Vertex is creating the first market of its kind and a unique token that combines trading, venture capital experience, and fair-value-based tokenization.","translator_type":"none","protected":false,"verified":false,"followers_count":19146,"friends_count":57

{"created_at":"Wed Nov 28 05:46:15 +0000 2018","id":1067655894073135106,"id_str":"1067655894073135106","text":"I really like the project. Prospects for the development of a purely cosmic scale. I wish the developers good healt\u2026 https:\/\/t.co\/oxB3GP2j5u","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":3154436241,"id_str":"3154436241","name":"muktamoni","screen_name":"mukta___moni","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":3998,"friends_count":4991,"listed_count":4,"favourites_count":5793,"statuses_count":6031,"created_at":"Fri Apr 10 20:38:53 +0000 2015","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":f

{"created_at":"Wed Nov 28 05:46:16 +0000 2018","id":1067655895616638976,"id_str":"1067655895616638976","text":"Bitcoin price news: How bitcoin may CRASH further in 2019 - Crypto CRISIS as BTC PLUMMETS - News Flash https:\/\/t.co\/HlXW0VPi4N","source":"\u003ca href=\"http:\/\/www.facebook.com\/twitter\" rel=\"nofollow\"\u003eFacebook\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":151435530,"id_str":"151435530","name":"ksb4328","screen_name":"ksb4744","location":"\ubd80\ucc9c","url":"http:\/\/koreantweeters.com\/ksb4328","description":"\ub09c \uc791\uc740\uc190","translator_type":"none","protected":false,"verified":false,"followers_count":66,"friends_count":281,"listed_count":3,"favourites_count":20,"statuses_count":39945,"created_at":"Thu Jun 03 12:01:47 +0000 2010","utc_offset":null,"time_zone":null,"geo_enabled":true,"lang":"ko","contri

{"created_at":"Wed Nov 28 05:46:16 +0000 2018","id":1067655897340473344,"id_str":"1067655897340473344","text":"RT @zibin: We are dedicated to supporting high growth #blockchain start-ups and the #FunctionX ecosystem. @PundiXLabs announced QEX fund in\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1072872451,"id_str":"1072872451","name":"Aaron Crypto$$\ud83d\ude0eDude","screen_name":"AarkonINDS3S","location":"Baja California, M\u00e9xico","url":null,"description":"CRYPTO is the only way now!! I belive in this gift and accept it. Thank you Lord.\ud83d\ude80\ud83d\ude80\ud83c\udfc4","translator_type":"none","protected":false,"verified":false,"followers_count":196,"friends_count":541,"listed_count":4,"favourites_count":3829,"st

{"created_at":"Wed Nov 28 05:46:17 +0000 2018","id":1067655899915767808,"id_str":"1067655899915767808","text":"RT @cripcy_info: \u3010\u4eba\u6c17\u8a18\u4e8b\u3011\nBitcoin\u30cd\u30c3\u30c8\u30ef\u30fc\u30af\u306f95%\u306e\u6e0b\u6ede\u306b\u9054\u3059\u308b\u3082\u3001\u624b\u6570\u6599\u306f\u5909\u308f\u3089\u305a\u3002\u53bb\u5e74\u3068\u306f\u9055\u3046\uff1f\n#\u30d3\u30c3\u30c8\u30b3\u30a4\u30f3 #\u624b\u6570\u6599 $BTC\nhttps:\/\/t.co\/TFX0FxuBL9 https:\/\/t.co\/7YM7bjGmE9","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1066587046137884673,"id_str":"1066587046137884673","name":"\u306a\u308b\u305b@782\u6b73","screen_name":"moneypleyer","location":"\u4e16\u754c\u4e2d\u306b\u6563\u3089\u3070\u3063\u3066\u308b","url":null,"des

{"created_at":"Wed Nov 28 05:46:17 +0000 2018","id":1067655901786583040,"id_str":"1067655901786583040","text":"RT @BTCTN: Fine Jewelry Dealer Birks Group Now Accepts Bitcoin https:\/\/t.co\/wJTnR7Usng https:\/\/t.co\/R6BDepZFY0","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":915625618997436416,"id_str":"915625618997436416","name":"Samy","screen_name":"Samy98684566","location":null,"url":null,"description":"CRYPTOCURRENCY INVESTOR\/ #BTC #BCH #XRP #XMR #TRX #ADA #XVG #ZEC #IOST #VEN #","translator_type":"none","protected":false,"verified":false,"followers_count":222,"friends_count":757,"listed_count":1,"favourites_count":2293,"statuses_count":1236,"created_at":"Wed Oct 04 17:12:13 +0000 2017","utc_offset":null,"time_zone":null,"geo

{"created_at":"Wed Nov 28 05:46:18 +0000 2018","id":1067655906014490624,"id_str":"1067655906014490624","text":"https:\/\/t.co\/cWXflgt43p Current Price of NANO. #NANO #nano #bitcoin Please ReTweet","source":"\u003ca href=\"http:\/\/ohiobitcoin.com\" rel=\"nofollow\"\u003eOhio Bitcoin 2014\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2640444888,"id_str":"2640444888","name":"Ohio Bitcoin \ud83d\udcca\ud83d\udcf0\ud83d\udcc8\ud83d\ude80","screen_name":"ohiobitcoin","location":"Moon","url":"http:\/\/OhioBitcoin.com","description":"Cryptocurrency Market Capitalization Price Index on #bitcoin & cryptocurrency. Real-time LIVE .#Bitcoin and related news. #HODL.","translator_type":"none","protected":false,"verified":false,"followers_count":12491,"friends_count":5452,"listed_count":583,"favourites_count":72121,"statuses_count":193365,"created_at

{"created_at":"Wed Nov 28 05:46:19 +0000 2018","id":1067655909659353088,"id_str":"1067655909659353088","text":"RT @VulcanoCoin: Vulcano ($VULC) to Present at the Public-Private Sector Social Enterprise Conference with UNESCO, the UNDP, the UN Migrati\u2026","source":"\u003ca href=\"https:\/\/vyper.io\" rel=\"nofollow\"\u003eVYPER Verification\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2781565590,"id_str":"2781565590","name":"Michael Baskakov","screen_name":"avator60","location":"Moscow","url":"https:\/\/in.xapo.com\/ref\/?c=2JMIB8P","description":"AVATOR","translator_type":"none","protected":false,"verified":false,"followers_count":105,"friends_count":1062,"listed_count":4,"favourites_count":130,"statuses_count":1833,"created_at":"Sun Aug 31 04:33:41 +0000 2014","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"ru","cont

{"created_at":"Wed Nov 28 05:46:19 +0000 2018","id":1067655910796001280,"id_str":"1067655910796001280","text":"Current Crypto Prices! \n\nBTC: $4030.72 USD\nETH: $114.98 USD\nLTC: $32.03 USD\nBCH: $196.86 USD\nXLM: $0.1521 USD\nDOGE:\u2026 https:\/\/t.co\/AT7qxBfD9e","source":"\u003ca href=\"https:\/\/twitter.com\/coinman23\" rel=\"nofollow\"\u003edetroit crypto\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":883051948827713537,"id_str":"883051948827713537","name":"J. Scardina","screen_name":"DetroitCrypto","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":10,"friends_count":1,"listed_count":3,"favourites_count":0,"statuses_count":498086,"created_at":"Thu Jul 06 19:56:05 +0000 2017","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enab

{"created_at":"Wed Nov 28 05:46:21 +0000 2018","id":1067655915757654016,"id_str":"1067655915757654016","text":"RT @Newstv_jp: IIT\u3068\u6771\u5927\u306e\u5354\u529b\u306e\u3082\u3068\u3001bitgrit\u304cWorldDataScienceForum\u3092\u958b\u50ac\uff01\u30d7\u30ed\u304c\u8a9e\u308b\u30c7\u30fc\u30bf\u30b5\u30a4\u30a8\u30f3\u30b9\u3068\u305d\u306e\u672a\u6765\nhttps:\/\/t.co\/3nN4530F2W\n#bitgrit  #AI #Blockchain #Datascie\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2244656766,"id_str":"2244656766","name":"manekineko2525","screen_name":"suteneko0709","location":null,"url":null,"description":"\ud83c\uddef\ud83c\uddf5\u8aa0\u5b9f\u3001\u4fe1\u983c\u3001\u7d14\u6f54\u3001\u539f\u5247\u3001\u53cb\u60c5\ud83d\udc31\ud83d\udc31 #\u4e0

{"created_at":"Wed Nov 28 05:46:21 +0000 2018","id":1067655916328169472,"id_str":"1067655916328169472","text":"RT @OracleOfCrypto: It's been awhile so I want to do a holiday giveaway. \n$2000 via BTC or whatever crypto you specify. \nJust retweet this\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":2536625304,"id_str":"2536625304","name":"Philip Drader","screen_name":"flipstix","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":2,"friends_count":43,"listed_count":0,"favourites_count":12,"statuses_count":29,"created_at":"Sat May 31 05:08:07 +0000 2014","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_tra

{"created_at":"Wed Nov 28 05:46:21 +0000 2018","id":1067655919029370880,"id_str":"1067655919029370880","text":"RT @rManoj1984: Wednesday 5 December, @kgauravITC will speak about \"The #Future of Financial Infrastructure \u2014 #Blockchain remoulding financ\u2026","source":"\u003ca href=\"https:\/\/mobile.twitter.com\" rel=\"nofollow\"\u003eTwitter Lite\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":4010776823,"id_str":"4010776823","name":"Ayojide.com","screen_name":"stayjid","location":"Ilorin, Nigeria","url":"http:\/\/info.ayojide.com\/","description":"Visit http:\/\/www.ayojide.com\/  and http:\/\/info.ayojide.com\/ #Synergy of #Ideas #Startups #Business #Solutions, #Digital #Promotions #Agric, Stuff That #Works","translator_type":"none","protected":false,"verified":false,"followers_count":4015,"friends_count":5001,"listed_count":73,"f

{"created_at":"Wed Nov 28 05:46:22 +0000 2018","id":1067655920430268416,"id_str":"1067655920430268416","text":"\ud83d\udd35 Blockchain, de la red de la informaci\u00f3n a la red del valor #Criptomonedas https:\/\/t.co\/8ArGoUC0fq","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1040021880,"id_str":"1040021880","name":"Tuiter Pro \ud83c\uddea\ud83c\uddf8","screen_name":"TuiterPro","location":"Comunidad de Madrid, Espa\u00f1a","url":"https:\/\/www.grandesmedios.com","description":"Los tuits al mejor nivel.","translator_type":"none","protected":false,"verified":false,"followers_count":179230,"friends_count":54920,"listed_count":1148,"favourites_count":3695,"statuses_count":1157129,"created_at":"Thu Dec 27 16:40:53 +0000 2012","utc_offset":null,"time_zone

{"created_at":"Wed Nov 28 05:46:22 +0000 2018","id":1067655922909011968,"id_str":"1067655922909011968","text":"RT @_Kevin_Pham: Calling all non-BTC supporters \"scammers\" is like calling all Trump supporters \"racists.\"","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":230428935,"id_str":"230428935","name":"Cloud 9","screen_name":"bataviavibe","location":"Honolulu, HI","url":null,"description":"Cryptotrader (Mind Body Spirit)","translator_type":"none","protected":false,"verified":false,"followers_count":232,"friends_count":583,"listed_count":3,"favourites_count":28167,"statuses_count":2935,"created_at":"Sat Dec 25 12:39:27 +0000 2010","utc_offset":null,"time_zone":null,"geo_enabled":true,"lang":"en","contributors_enabled":false,"is_

{"created_at":"Wed Nov 28 05:46:23 +0000 2018","id":1067655925366849536,"id_str":"1067655925366849536","text":"BTC\u3069\u77ed\u671f\u30ed\u30f3\u30b0\uff01 https:\/\/t.co\/PDtzCC1Tp3","display_text_range":[0,10],"source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":4914710720,"id_str":"4914710720","name":"\u30a2\u30ea\u30da\u30fc","screen_name":"artkfm002","location":"\u6771\u4eac","url":"https:\/\/line.me\/R\/ti\/p\/%40aaq7626r","description":"22\u6b73\/\u4eee\u60f3\u901a\u8ca8FX\u30c8\u30ec\u30fc\u30c0\u30fc\/\u6700\u9ad8\u65e5\u52291500%\/\u5927\u5206\u770c\u306e\u7530\u820e\u2192\u6771\u4eac\u2192\u5317\u4eac\u306b\u7559\u5b66\u2192\u6771\u4eac\/\u8da3\u5473\u306f\u9b5a\u91e3\u308a\u3001\u6620\u753b\u9451\u8cde\u3001\u56fd\u5185\u3001

{"created_at":"Wed Nov 28 05:46:24 +0000 2018","id":1067655931587162112,"id_str":"1067655931587162112","text":"RT @Venus_Energy: https:\/\/t.co\/SYl0l2zsBp\n\n#venus #venusenergy #venuscoin #blockchain #etherscan #bitcoin #ethereum    #newcryptocurrency #\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1063766814373675010,"id_str":"1063766814373675010","name":"Anthony Mason","screen_name":"Anthony13719826","location":"Miami, FL","url":"http:\/\/www.luno.com","description":"Miner at bitcoin news & cryptocurrency trading investment bank- forex trader-\nStudy Digital marketing at National university of Malaysia","translator_type":"none","protected":false,"verified":false,"followers_count":94,"friends_count":251,"listed_count":0

{"created_at":"Wed Nov 28 05:46:25 +0000 2018","id":1067655933197725696,"id_str":"1067655933197725696","text":"Bitcoin SV Enters Top Ten \u2013 News Crypto https:\/\/t.co\/bp0hxjhxdz","source":"\u003ca href=\"https:\/\/paper.li\" rel=\"nofollow\"\u003ePaper.li\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":900995460399280128,"id_str":"900995460399280128","name":"K01N_org","screen_name":"k01n_org","location":"Online","url":"http:\/\/www.K01N.org","description":"the world has changed","translator_type":"none","protected":false,"verified":false,"followers_count":23,"friends_count":56,"listed_count":0,"favourites_count":3,"statuses_count":463,"created_at":"Fri Aug 25 08:17:12 +0000 2017","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"profile_background_color":"000000","p

{"created_at":"Wed Nov 28 05:46:25 +0000 2018","id":1067655934594342912,"id_str":"1067655934594342912","text":"\u3053\u308c\u3002\u6628\u65e5\u306e\u30c4\u30a4\u30fc\u30c8\u3002\n\u7b2c1\u95a2\u9580\u7a81\u7834\u3057\u305f\u3051\u3093\u5f8c\u306f\u9577\u3044\u6642\u9593\u8db3\u3067\u9006\u4e09\u5c0a\u4f5c\u308b\u3053\u3068\u671f\u5f85\u2b50\ufe0f\n\n\u4eca\u304b\u3089\u5c11\u3057\u4e0b\u3052\u2935\ufe0f\u6765\u308b\u3068\u601d\u3046\u3051\u3069\u4eca\u306e\u3068\u3053\u308d\u5f8c\u306f\u304a\u7d75\u63cf\u304d\u901a\u308a\u306e\u30a4\u30e1\u30fc\u30b8\u3002\n\u8010\u3048\u305f\u3089\u9006\u4e09\u5c0a\u4f5c\u3063\u3066\u4e0a\u2191\u3088\u3002\n\nBTC\u3082\u30461\u6bb5\u4e0a\u304c\u308b\u306e\u5f85\u3063\u3068\u30fc\u3088\u2606\u2026 https:\/\/t.co\/gVKc0mts4H","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"i

{"created_at":"Wed Nov 28 05:46:25 +0000 2018","id":1067655934825119745,"id_str":"1067655934825119745","text":"Bitcoin (BTC) Bulls Are Back in the Game! https:\/\/t.co\/gjpqE53371 #bitcoinnews #bitcoinbulls https:\/\/t.co\/DMUfLtEvRM","display_text_range":[0,92],"source":"\u003ca href=\"https:\/\/markethive.com\" rel=\"nofollow\"\u003eMarketHive Oneall\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":198429471,"id_str":"198429471","name":"John Anderson","screen_name":"hpsteam","location":"Morwell,Victoria, Australia","url":"http:\/\/infinity-pay-plan.com","description":"I use to drive steam engines for a living! Now i am part of the MARKETHIVE Alpha Founders Team! Join us and you to can be FREE!","translator_type":"none","protected":false,"verified":false,"followers_count":949,"friends_count":2365,"listed_count":19,"favourites_count":2,"st

{"created_at":"Wed Nov 28 05:46:26 +0000 2018","id":1067655937308135425,"id_str":"1067655937308135425","text":"The future is clear.\nHodl my brothers!\nThe World Trade Organization (WTO) released a reportonblockchaintechnology\u2019s\u2026 https:\/\/t.co\/eLpbypiqcZ","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1027302559726399488,"id_str":"1027302559726399488","name":"Thunderms100","screen_name":"thunderms100","location":"Minneapolis, MN","url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":6,"friends_count":40,"listed_count":0,"favourites_count":35,"statuses_count":8,"created_at":"Wed Aug 08 21:16:31 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":

{"created_at":"Wed Nov 28 05:46:26 +0000 2018","id":1067655938692313088,"id_str":"1067655938692313088","text":"\u3010\u901f\u5831\u3011\u30d3\u30c3\u30c8\u30b3\u30a4\u30f3\u6025\u9a30\u3067\uff14\uff16\u4e07\u5186&amp;\uff14\uff10\uff10\uff10\u30c9\u30eb\u7a81\u7834\uff01\uff01\uff01(\u203b\u753b\u50cf\u3042\u308a)\u3000\u3000\u3000\u3000\u3000\u3000#\u4eee\u60f3\u901a\u8ca8\u3000$BTC https:\/\/t.co\/dhdTGFnjaM https:\/\/t.co\/KxsootpW50","display_text_range":[0,80],"source":"\u003ca href=\"http:\/\/publicize.wp.com\/\" rel=\"nofollow\"\u003eWordPress.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":948530405157711873,"id_str":"948530405157711873","name":"\u732a\u7a81\u731b\u9032\u306e\u7a81\u6483\u968a\u9577","screen_name":"inosisitaichou","location":null,"url":null,"description":"\u76f8\u4e92","translator_type":"none","protected":fal

{"created_at":"Wed Nov 28 05:46:26 +0000 2018","id":1067655940390961153,"id_str":"1067655940390961153","text":"CEO Blockchain Intelligence Group: Fundamental Bitcoin Masih Utuh Meskipun Harga\u00a0Rendah https:\/\/t.co\/Zz1dD2Sacp","source":"\u003ca href=\"http:\/\/publicize.wp.com\/\" rel=\"nofollow\"\u003eWordPress.com\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":279019352,"id_str":"279019352","name":"Tabungan Internet","screen_name":"fxProfitsMaster","location":null,"url":"http:\/\/www.TabunganInternet.com","description":"Tabungan Syariah, Tabungan Investasi, Investasi, Syariah - Usaha kemitraan bagi hasil syariah","translator_type":"none","protected":false,"verified":false,"followers_count":33,"friends_count":0,"listed_count":1,"favourites_count":1,"statuses_count":7467,"created_at":"Fri Apr 08 12:37:06 +0000 2011","utc_offset":nul

{"created_at":"Wed Nov 28 05:46:27 +0000 2018","id":1067655944304238592,"id_str":"1067655944304238592","text":"RT @finafex: Our Airdrop is live \n\nJoin now at https:\/\/t.co\/bNgqETJntC  and earn up to 370 FNX Tokens \n\n#airdrop #ico #bitcoin #cryptocurre\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1010238080945664001,"id_str":"1010238080945664001","name":"Alexander","screen_name":"Alexand93292616","location":"Uganda","url":null,"description":"A digital lover and btc trader.","translator_type":"none","protected":false,"verified":false,"followers_count":57,"friends_count":195,"listed_count":0,"favourites_count":45,"statuses_count":56,"created_at":"Fri Jun 22 19:08:22 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang"

{"created_at":"Wed Nov 28 05:46:27 +0000 2018","id":1067655944849506304,"id_str":"1067655944849506304","text":"This 4H candle is pretty vital in determining what action I am going to take over the next few weeks. Would love a\u2026 https:\/\/t.co\/pdKK2RMjY4","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":37238588,"id_str":"37238588","name":"Brad Hammack","screen_name":"BradHammack","location":"Paducah, KY","url":null,"description":"Cryptotoad\/Music Educator\/Phreshwadda bassist\/","translator_type":"none","protected":false,"verified":false,"followers_count":127,"friends_count":451,"listed_count":1,"favourites_count":505,"statuses_count":3386,"created_at":"Sat May 02 17:18:59 +0000 2009","utc_offset":null,"time_zone":null,"geo_enabl

{"created_at":"Wed Nov 28 05:46:29 +0000 2018","id":1067655950243373056,"id_str":"1067655950243373056","text":"RT @DiarioBitcoin: Lo que Vitalik Buterin no dijo cuando reneg\u00f3 de algunos usos de Blockchain https:\/\/t.co\/BjjmSshOQr https:\/\/t.co\/7Py1GBnC\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1010030809829560320,"id_str":"1010030809829560320","name":"extenuado","screen_name":"extenuado1","location":null,"url":null,"description":"Cansado, decepcionado, triste por la vida","translator_type":"none","protected":false,"verified":false,"followers_count":33,"friends_count":110,"listed_count":0,"favourites_count":112,"statuses_count":120,"created_at":"Fri Jun 22 05:24:45 +0000 2018","utc_offset":null,"time_zone":null,"

{"created_at":"Wed Nov 28 05:46:30 +0000 2018","id":1067655956899672064,"id_str":"1067655956899672064","text":"RT @Moozicore: The music streaming revolution is on! \nIn difficult times for the #crypto community it is essential to work hard on new proj\u2026","source":"\u003ca href=\"https:\/\/mobile.twitter.com\" rel=\"nofollow\"\u003eTwitter Lite\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":956103313195384833,"id_str":"956103313195384833","name":"Romeotom","screen_name":"Romeotom360","location":null,"url":null,"description":"#Bitcoin #Altcoin #Blockchain #Cryptocurrencies #Investment #ICO #Cryptonews #Cryptoregulation\n#Follow_me_and_back_you","translator_type":"none","protected":false,"verified":false,"followers_count":8859,"friends_count":9518,"listed_count":16,"favourites_count":6933,"statuses_count":6516,"created_at":"Wed Jan 24 0

{"created_at":"Wed Nov 28 05:46:31 +0000 2018","id":1067655957889466368,"id_str":"1067655957889466368","text":"RT @menapayio: If you are CURIOUS about the secrets to what to look for when investing in an ICO, check this article out: \nhttps:\/\/t.co\/Zgx\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":3154055740,"id_str":"3154055740","name":"jadalily","screen_name":"lily_jada","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":3851,"friends_count":4860,"listed_count":4,"favourites_count":4775,"statuses_count":4734,"created_at":"Fri Apr 10 16:34:36 +0000 2015","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":fal

{"created_at":"Wed Nov 28 05:46:31 +0000 2018","id":1067655960917872640,"id_str":"1067655960917872640","text":"RT @blockchainwtf: Tone Vays is more than just a Bitcoin enthusiast with a YouTube channel. The former Wall Street analyst is an example of\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":860730680581406720,"id_str":"860730680581406720","name":"Disruption Joe","screen_name":"DisruptionJoe","location":"Chicago, IL","url":"https:\/\/www.disruptionjoe.com","description":"Founder, Chicago Blockchain Project http:\/\/chicagoblockchainproject.com\nhttp:\/\/blockchaininchicago.com\n#buidl #ChicagoHodl #fatprotocols","translator_type":"none","protected":false,"verified":false,"followers_count":1545,"friends_count":1886,"liste

{"created_at":"Wed Nov 28 05:46:32 +0000 2018","id":1067655962306232325,"id_str":"1067655962306232325","text":"Bitcoin Lane #BeatlesSongsForMillennials","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":954847957462978560,"id_str":"954847957462978560","name":"Roger Thornhill","screen_name":"RogerThorn_hill","location":"United States","url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":581,"friends_count":992,"listed_count":1,"favourites_count":3070,"statuses_count":5431,"created_at":"Sat Jan 20 22:47:48 +0000 2018","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"profile_background_color":"F5F8FA","profile_background_image_url":"

{"created_at":"Wed Nov 28 05:46:32 +0000 2018","id":1067655964814381057,"id_str":"1067655964814381057","text":"\u3010\u901f\u5831\u3011\u3010\u901f\u5831\u3011\u30d3\u30c3\u30c8\u30b3\u30a4\u30f345\u4e07\u5186\u7a81\u7834\uff01\u30ab\u30c1\u4e0a\u3052\uff77\uff80\u2501\u2501\u2501(\uff9f\u2200\uff9f)\u2501\u2501\u2501!!\u3010BTC\u3011\u2192\u8a73\u7d30\u306f\u30b3\u30c1\u30e9https:\/\/t.co\/SqFxFQlwjo\u3000by\u4eee\u60f3\u901a\u8ca8\u3067\u7a3c\u3050 https:\/\/t.co\/l7UhhzeqIv","display_text_range":[0,85],"source":"\u003ca href=\"https:\/\/ifttt.com\" rel=\"nofollow\"\u003eIFTTT\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":716528503995256833,"id_str":"716528503995256833","name":"\u4eee\u60f3\u901a\u8ca8\u3067\u7a3c\u3050\uff01","screen_name":"2chnoiz","location":null,"url":"http:\/\/money.lifetrivia.net\/","description":"\u4eee\u60f3\u

{"created_at":"Wed Nov 28 05:46:33 +0000 2018","id":1067655969302290432,"id_str":"1067655969302290432","text":"RT @coindesk: 2\/ Bloomberg reported earlier Tuesday that, according to sources, Nasdaq is still pursuing the launch of bitcoin futures in 2\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":905835030051684356,"id_str":"905835030051684356","name":"Doctor-crypto","screen_name":"upgradecrypto","location":null,"url":null,"description":"*crypto analyst\n\n*crypto adict\n\n*Contact me for private analyse\n\n*Share  the New revolution of crypto \ud83d\udcbb\ud83d\udcb0\ud83d\udc4d","translator_type":"none","protected":false,"verified":false,"followers_count":121,"friends_count":858,"listed_count":0,"favourites_count":1060,"

{"created_at":"Wed Nov 28 05:46:34 +0000 2018","id":1067655973479874560,"id_str":"1067655973479874560","text":"RT @zibin: We are dedicated to supporting high growth #blockchain start-ups and the #FunctionX ecosystem. @PundiXLabs announced QEX fund in\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":945417194468409346,"id_str":"945417194468409346","name":"Joseph(x)","screen_name":"Jvdwesterlo","location":"Nederland","url":null,"description":"Family, Exercise, Crypto. Not financial advice. $NPXS, $BNB, $MFT, $VET, $BTC.","translator_type":"none","protected":false,"verified":false,"followers_count":257,"friends_count":85,"listed_count":0,"favourites_count":792,"statuses_count":528,"created_at":"Mon Dec 25 22:13:18 +0000 2017","u

{"created_at":"Wed Nov 28 05:46:35 +0000 2018","id":1067655976487198720,"id_str":"1067655976487198720","text":"RT @JBTheCryptoKing: @APompliano Couldn't have said it better myself.\n\nBTC will be completely obsolete by the next halving or it'll be wort\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/iphone\" rel=\"nofollow\"\u003eTwitter for iPhone\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":954506792767229952,"id_str":"954506792767229952","name":"Joe Altz","screen_name":"AltzJoe","location":"Michigan, USA","url":null,"description":"DYOR | HODL | \ud83d\ude80 | Crypto | Love the game |","translator_type":"none","protected":false,"verified":false,"followers_count":95,"friends_count":103,"listed_count":1,"favourites_count":3051,"statuses_count":754,"created_at":"Sat Jan 20 00:12:08 +0000 2018","utc_offset":null,"time_zon

{"created_at":"Wed Nov 28 05:46:36 +0000 2018","id":1067655980752781312,"id_str":"1067655980752781312","text":"RT @Shirastweet: Excited to announce I signed a book deal. Expected release date Summer 2019 #cybersecurity #CyberWarfare #cloudsecurity #A\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":622202083,"id_str":"622202083","name":"Georges Wedji","screen_name":"Gwedji","location":null,"url":null,"description":"(Security,BigData,Cloud,Network,Software,Databases,IoT,Telecoms,Innovation)","translator_type":"none","protected":false,"verified":false,"followers_count":584,"friends_count":386,"listed_count":578,"favourites_count":9161,"statuses_count":8623,"created_at":"Fri Jun 29 20:08:22 +0000 2012","utc_offset":null,"time_zone":null,"geo_enable

{"created_at":"Wed Nov 28 05:46:37 +0000 2018","id":1067655982979981312,"id_str":"1067655982979981312","text":"RT @VulcanoCoin: Vulcano is redefining the role of cryptocurrency in Research, our small size packs big punches. $vulc #crypto #vulcano #re\u2026","source":"\u003ca href=\"https:\/\/vyper.io\" rel=\"nofollow\"\u003eVYPER Verification\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":1064421531730829313,"id_str":"1064421531730829313","name":"Polina","screen_name":"Polina83931228","location":null,"url":null,"description":"# \u0414\u0435\u0441\u0430\u043d\u0442 #\u0434\u0435\u0441\u0430\u043d\u0442\u0438\u0440\u043e\u0432\u0430\u043d\u0438\u0435 # Welleth #\u0412\u0435\u043a\u0441 #\u0418\u041a\u041e","translator_type":"none","protected":false,"verified":false,"followers_count":10,"friends_count":58,"listed_count":0,"favourites_count"

{"created_at":"Wed Nov 28 05:46:37 +0000 2018","id":1067655985756426240,"id_str":"1067655985756426240","text":"RT @crypto: JUST IN: The head of the SEC says a lack of investor protections makes it unlikely that his agency will approve a Bitcoin ETF a\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":950387968086192128,"id_str":"950387968086192128","name":"Jon Zenaro","screen_name":"Cryptos_101","location":"Republic of the Philippines","url":null,"description":"ArtWork | Graphic Designer | Online Entrepreneur | Crypto Airdrop\/Bounty | Trader Bitcoin\/CryptoCurrency #blockchain #cryptoworld","translator_type":"none","protected":false,"verified":false,"followers_count":311,"friends_count":1167,"listed_count":1,"favourites_count"

{"created_at":"Wed Nov 28 05:46:37 +0000 2018","id":1067655986695950337,"id_str":"1067655986695950337","text":"The latest bitcoin plunge is making its fans loopy https:\/\/t.co\/oeFDe38E1u","source":"\u003ca href=\"http:\/\/www.facebook.com\/twitter\" rel=\"nofollow\"\u003eFacebook\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":151435530,"id_str":"151435530","name":"ksb4328","screen_name":"ksb4744","location":"\ubd80\ucc9c","url":"http:\/\/koreantweeters.com\/ksb4328","description":"\ub09c \uc791\uc740\uc190","translator_type":"none","protected":false,"verified":false,"followers_count":66,"friends_count":281,"listed_count":3,"favourites_count":20,"statuses_count":39946,"created_at":"Thu Jun 03 12:01:47 +0000 2010","utc_offset":null,"time_zone":null,"geo_enabled":true,"lang":"ko","contributors_enabled":false,"is_translator":false,"profile

{"created_at":"Wed Nov 28 05:46:38 +0000 2018","id":1067655990277885953,"id_str":"1067655990277885953","text":"RT @cedium_ceu: Good news for the chefs of the digital age! You can let the whole world discover your cooking skills by boosting your cooki\u2026","source":"\u003ca href=\"http:\/\/twitter.com\/#!\/download\/ipad\" rel=\"nofollow\"\u003eTwitter for iPad\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":181530669,"id_str":"181530669","name":"Token","screen_name":"Jeevan9191","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":10316,"friends_count":6770,"listed_count":20,"favourites_count":11908,"statuses_count":15142,"created_at":"Sun Aug 22 12:42:15 +0000 2010","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_t

{"created_at":"Wed Nov 28 05:46:38 +0000 2018","id":1067655990433202177,"id_str":"1067655990433202177","text":"S\u0131rf #bitcoin\nVe #crypto lar i\u00e7in a\u00e7m\u0131\u015ft\u0131m by sayfay\u0131 1 y\u0131l olmu\u015f zaman ne \u00e7abuk ge\u00e7iyor hala zengin olamamam\u270c\nTwit\u2026 https:\/\/t.co\/Rk34XEPpks","display_text_range":[0,140],"source":"\u003ca href=\"http:\/\/twitter.com\/download\/android\" rel=\"nofollow\"\u003eTwitter for Android\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":932356484075278337,"id_str":"932356484075278337","name":"Ena","screen_name":"EnaYlmz","location":"\u0130stanbul, T\u00fcrkiye ULUDA\u011e UNIV.","url":null,"description":"Kriptoda topuk sesleri !\n\n\nKisa,\u00f6z,net.","translator_type":"none","protected":false,"verified":false,"followers_count":324,"friends_count":188,"listed_count":8,"fa

{"created_at":"Wed Nov 28 05:46:41 +0000 2018","id":1067656003435425792,"id_str":"1067656003435425792","text":"\u30d3\u30c3\u30c8\u30b3\u30a4\u30f3\u3092\u5b89\u5168\u306b\u5897\u3084\u3059\u70ba\u306b\u7c21\u5358\u306a\u65b9\u6cd5\u306f\u30de\u30a4\u30cb\u30f3\u30b0\u3067\u3059\u3002\u8a73\u7d30\u306f\u3053\u3061\u3089\u3092\u30a2\u30af\u30bb\u30b9\u2192\u2192\u3000\u3000https:\/\/t.co\/QAw2AtxDRF\u3000\u30a6\u30a9\u30ec\u30c3\u30c8\u306b\u534a\u6708\u3054\u3068\u306b\u30de\u30a4\u30cb\u30f3\u30b0\u53ce\u76ca\u304c\u9084\u5143\u3055\u308c\u307e\u3059\uff01\u3000#\u30d3\u30c3\u30c8\u30b3\u30a4\u30f3\u3000#BTC\u3000#\u30ea\u30c3\u30d7\u30eb\u3000\u3000#\u30a4\u30fc\u30b5\u30ea\u30a2\u30e0","source":"\u003ca href=\"http:\/\/twittbot.net\/\" rel=\"nofollow\"\u003etwittbot.net\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":713602948249464832

{"created_at":"Wed Nov 28 05:46:42 +0000 2018","id":1067656004014227457,"id_str":"1067656004014227457","text":"I assume the reason the event-stream\/copay hack only stole your account details if the balance was over 100 bitcoin\u2026 https:\/\/t.co\/TplkFfnNvm","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":true,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":15453,"id_str":"15453","name":"Laurie Voss","screen_name":"seldo","location":"San Francisco, CA","url":"http:\/\/seldo.com","description":"I like websites. Co-founder\/COO of @npmjs, started http:\/\/lgbtq.technology. He\/him. \ud83c\uddf9\ud83c\uddf9\ud83c\udff3\ufe0f\u200d\ud83c\udf08","translator_type":"none","protected":false,"verified":false,"followers_count":22027,"friends_count":1963,"listed_count":857,"favourites_count":82768,"statuses_coun

{"created_at":"Wed Nov 28 05:46:44 +0000 2018","id":1067656013094895616,"id_str":"1067656013094895616","text":"RT @menapayio: Did you know? @MenaPay is using #blockchain technology to create a single currency that follows the Islamic finance practice\u2026","source":"\u003ca href=\"http:\/\/twitter.com\" rel=\"nofollow\"\u003eTwitter Web Client\u003c\/a\u003e","truncated":false,"in_reply_to_status_id":null,"in_reply_to_status_id_str":null,"in_reply_to_user_id":null,"in_reply_to_user_id_str":null,"in_reply_to_screen_name":null,"user":{"id":3154055740,"id_str":"3154055740","name":"jadalily","screen_name":"lily_jada","location":null,"url":null,"description":null,"translator_type":"none","protected":false,"verified":false,"followers_count":3851,"friends_count":4860,"listed_count":4,"favourites_count":4776,"statuses_count":4735,"created_at":"Fri Apr 10 16:34:36 +0000 2015","utc_offset":null,"time_zone":null,"geo_enabled":false,"lang":"en","contributors_enabled":false,"is_translator":false,"

ProtocolError: ('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))

#### Reading and storing in a Dataframe
- Lets read the the saved file as defined above in the location "./live_stream_tweets.json" in a read mode line by line.

In [4]:
# Initialise an empyt list to store the tweets as they get read line by line
tweets_data = []

# open the stored file in a read mode\n",
tweets_file = open('./live_stream_tweets.json',"r")
#tweets_file

# for loop to iterate over the tweets_file and using json.loads\n",
# json.load is for a file and json.loads is for a string, here will use loads as we have already read the file above.\n",

for line in tweets_file:
    try:
        #print(line)
        tweet = json.loads(line)
        #print(tweet)
        tweets_data.append(tweet)
    except:
        continue

#### Print the length of tweets extracted

In [5]:
print(len(tweets_data))

40235


#### Build a DataFrame of tweet texts and other columns
----
- The columns I need in my dataframe are : creaed_at, id, text, lang
- *lang column will be dropped after filtering only english language tweets*
- Tweets length can be derived from the tweets text length which is max 140 characters.
- Next, is the spearation of date and time, as it would be useful in doing time-series like counts of tweets in a day or an hour.
- Lastly, reset the index after the english tweets filtering and drop the index and language column."

In [ ]:
def tweets_to_data_frame(tweets):
    df = pd.DataFrame(data=tweets, columns=['created_at','id','text','lang'])
    # Adding another column for the length of the tweet
    #df['Tweet_len'] = np.array([len(x) for x in df.text])
    df = df[df.lang == 'en']
    df['created_at'] = pd.to_datetime(df['created_at'])
    df['timestamp'] = pd.to_datetime(df['created_at'])
    df['created_at'] = df['created_at'].astype('object')
    df['Date'] = df['created_at'].dt.date
    df['Timestamp'] = df['timestamp'].dt.time
    df.reset_index()
    df.drop(['lang'], axis=1,inplace=True)
    df.drop(['index'], axis=1,inplace=True) 
    

In [25]:
len(df.text)

40235

In [32]:
tweets_to_data_frame(tweets_data)

index           created_at            id  \
0          1  2018-11-28 05:45:43  1.067656e+18   
1          2  2018-11-28 05:45:45  1.067656e+18   
2          3  2018-11-28 05:45:46  1.067656e+18   
3          4  2018-11-28 05:45:46  1.067656e+18   
4          5  2018-11-28 05:45:46  1.067656e+18   
5          6  2018-11-28 05:45:47  1.067656e+18   
6          7  2018-11-28 05:45:47  1.067656e+18   
7          8  2018-11-28 05:45:48  1.067656e+18   
8          9  2018-11-28 05:45:48  1.067656e+18   
9         10  2018-11-28 05:45:48  1.067656e+18   
10        11  2018-11-28 05:45:49  1.067656e+18   
11        13  2018-11-28 05:45:50  1.067656e+18   
12        17  2018-11-28 05:45:52  1.067656e+18   
13        18  2018-11-28 05:45:52  1.067656e+18   
14        20  2018-11-28 05:45:53  1.067656e+18   
15        22  2018-11-28 05:45:53  1.067656e+18   
16        24  2018-11-28 05:45:53  1.067656e+18   
17        25  2018-11-28 05:45:54  1.067656e+18   
18        27  2018-11-28 05:45:55  1.067656e+18   
19        28  2018-11-28 05:45:57  1.067656e+18   
20        30  2018-11-28 05:45:58  1.067656e+18   
21        31  2018-11-28 05:45:58  1.067656e+18   
22        32  2018-11-28 05:45:58  1.067656e+18   
23        33  2018-11-28 05:45:58  1.067656e+18   
24        34  2018-11-28 05:45:58  1.067656e+18   
25        35  2018-11-28 05:45:58  1.067656e+18   
26        36  2018-11-28 05:45:59  1.067656e+18   
27        37  2018-11-28 05:45:59  1.067656e+18   
28        38  2018-11-28 05:46:00  1.067656e+18   
29        40  2018-11-28 05:46:00  1.067656e+18   
...      ...                  ...           ...   
32922  40197  2018-11-29 04:16:09  1.067996e+18   
32923  40198  2018-11-29 04:16:09  1.067996e+18   
32924  40199  2018-11-29 04:16:09  1.067996e+18   
32925  40200  2018-11-29 04:16:09  1.067996e+18   
32926  40202  2018-11-29 04:16:10  1.067996e+18   
32927  40203  2018-11-29 04:16:11  1.067996e+18   
32928  40204  2018-11-29 04:16:12  1.067996e+18   
32929  40205  2018-11-29 04:16:13  1.067996e+18   
32930  40206  2018-11-29 04:16:13  1.067996e+18   
32931  40208  2018-11-29 04:16:16  1.067996e+18   
32932  40210  2018-11-29 04:16:17  1.067996e+18   
32933  40211  2018-11-29 04:16:18  1.067996e+18   
32934  40212  2018-11-29 04:16:18  1.067996e+18   
32935  40214  2018-11-29 04:16:19  1.067996e+18   
32936  40215  2018-11-29 04:16:20  1.067996e+18   
32937  40216  2018-11-29 04:16:20  1.067996e+18   
32938  40217  2018-11-29 04:16:21  1.067996e+18   
32939  40218  2018-11-29 04:16:22  1.067996e+18   
32940  40219  2018-11-29 04:16:25  1.067996e+18   
32941  40220  2018-11-29 04:16:26  1.067996e+18   
32942  40221  2018-11-29 04:16:26  1.067996e+18   
32943  40224  2018-11-29 04:16:28  1.067996e+18   
32944  40225  2018-11-29 04:16:28  1.067996e+18   
32945  40226  2018-11-29 04:16:31  1.067996e+18   
32946  40227  2018-11-29 04:16:31  1.067996e+18   
32947  40228  2018-11-29 04:16:31  1.067996e+18   
32948  40229  2018-11-29 04:16:33  1.067996e+18   
32949  40230  2018-11-29 04:16:33  1.067996e+18   
32950  40231  2018-11-29 04:16:33  1.067996e+18   
32951  40234  2018-11-29 04:16:33  1.067996e+18   

                                                    text lang  \
0      RT @CoinPaymentsNET: We are happy to announce ...   en   
1      RT @UserExperienceU: Speeding up the Wrong Wor...   en   
2      RT @mishalederman: ✍🏽@Forbes article names #TR...   en   
3      RT @SmartContracter: This is one of my favouri...   en   
4      Love to know more click the link bellow :)\n#F...   en   
5      RT @Builderium: We are working on the engine o...   en   
6      RT @DeepCloud_AI: We are so close to presentin...   en   
7      RT @buying_com: We are very excited to announc...   en   
8      RT @ToneVays: Let the #Bitcoin CAPITULATION Be...   en   
9      Bitcoin LONG https://t.co/WW9lYxpNHF #bitcoin ...   en   
10     RT @bcb_atm: Landmark Research from IIT Bombay...   en   
11     India's leading educational institute, SP Jain...   en   

In [33]:
df.shape

(32952, 7)

#### Create a new dataframe

In [41]:
cols = {'Date' : [], 'Timestamp' : [], 'Id' : [], 'Text' : [] }
tweets_df = pd.DataFrame(data=cols)

# Copying the contents the dataframe values into a new dataframe.
tweets_df['Date'] = df['Date']
tweets_df['Timestamp'] = df['Timestamp']
tweets_df['Id'] = df['id']
tweets_df['Text'] = df['text']

# Print the head
tweets_df.head()

Date Timestamp            Id  \
1  2018-11-28  05:45:43  1.067656e+18   
2  2018-11-28  05:45:45  1.067656e+18   
3  2018-11-28  05:45:46  1.067656e+18   
4  2018-11-28  05:45:46  1.067656e+18   
5  2018-11-28  05:45:46  1.067656e+18   

                                                Text  
1  RT @CoinPaymentsNET: We are happy to announce ...  
2  RT @UserExperienceU: Speeding up the Wrong Wor...  
3  RT @mishalederman: ✍🏽@Forbes article names #TR...  
4  RT @SmartContracter: This is one of my favouri...  
5  Love to know more click the link bellow :)\n#F...

In [42]:
tweets_df.shape

(32952, 4)

In [43]:
tweets_df.dtypes

Date          object
Timestamp     object
Id           float64
Text          object
dtype: object